android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedNotificationRequest.java

```diff
@@ -0,0 +1,54 @@
+package versioned.host.exp.exponent.modules.universal.notifications;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import expo.modules.notifications.notifications.interfaces.NotificationTrigger;
+import expo.modules.notifications.notifications.model.NotificationContent;
+import expo.modules.notifications.notifications.model.NotificationRequest;
+import host.exp.exponent.kernel.ExperienceId;
+
+public class ScopedNotificationRequest extends NotificationRequest {
+  // We store String instead of ExperienceId, cause ScopedNotificationRequest must be serializable.
+  private String mExperienceIdString;
+
+  public ScopedNotificationRequest(String identifier, NotificationContent content, NotificationTrigger trigger, @Nullable ExperienceId experienceId) {
+    super(identifier, content, trigger);
+    if (experienceId == null) {
+      mExperienceIdString = "";
```

Comment:
Is there any benefit to saving `mExperienceIdString = ""`? Couldn't we use `null`? It looks like you guard against it in `checkIfBelongsToExperience` nonetheless. 👍